crates/turborepo-lib/src/daemon/client.rs

```diff
@@ -29,7 +25,8 @@ impl<T: Clone> DaemonClient<T> {
         let _ret = self
             .client
             .hello(proto::HelloRequest {
-                version: get_version().to_string(),
+                version: proto::VERSION.to_string(),
+                supported_version_range: proto::VersionRange::Minor.into(),
```

Comment:
Should this be in a constant somewhere with a comment explaining when/why it should change? Do we expect it to always be `Minor`?

Reply:
I think this will likely be the default. It is similar to the npm caret which is by far the most popular version specifier. I have added a comment